### PR TITLE
Ellenmarie 63-a

### PR DESCRIPTION
**Pull request by ellenmarie**
Here are all of my edits/questions for -A.

### CODE REVIEW

File: sp800-63a/sec1_2_introduction.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ _This section is informative._
...
 
-There are also instances where it is desirable for a relying party (RP) to know something about a user executing a transaction, but not know the real-life identity of the user.  For example, in order to maintain integrity of the service, it may be desirable to know the home ZIP Code of a user for purposes of census taking or petitioning an elected official but where it is not necessary or desirable to know the underlying identity of the person. IALs provide a method for expressing the level of assurance (LOA) associated with attributes established by the CSP during the proofing process.
+There are also instances where it is desirable for a relying party (RP) to know something about a subject executing a transaction, but not know the real-life identity. For example, it may be desirable to only know the home ZIP Code of a subscriber for purposes of census taking, or petitioning an elected official. In both instances, the zip code is sufficient to deliver the service; it is not necessary or desirable to know the underlying identity of the person.
```
Comment (jimfenton) [inline sp800-63a/sec1_2_introduction.md]: zip -> ZIP (in last sentence)

File: sp800-63a/sec1_2_introduction.md [hunk 2]
```diff
@@ -46,12 +46,12 @@ When a subject is identity proofed, the expected steps are:
...
+At IAL2 and IAL3, pseudonymity in federated environments is enabled by limiting the number of attributes sent from the CSP to the RP, or the way they are presented. For example, if an RP needs a valid birthdate but no other personal details, the RP should leverage a CSP to request just the birthdate of the subscriber. It is preferred for the RP to ask the CSP for an attribute reference. For example, if a RP needs to know if a subscriber is older than 18 they should request a boolean value, not the entire birthdate in order to evaluate age. Users who maintain and IAL3 identity should be able to use their CSP for lower-level and pseudonymous transactions, giving them the added benefit of being able to assert already-verified attributes as part of these transactions. Since the individual will have undergone an identity proofing process at enrollment, interactions with the CSP may not necessarily be pseudonymous.
```
Comment (jimfenton) [inline sp800-63a/sec1_2_introduction.md]: a RP -> an RP (following "For example,")

File: sp800-63a/sec4_ial.md [hunk 3]
```diff
@@ -258,11 +258,11 @@ _This section is informative._
 
 Requirement | IAL1 | IAL2 | IAL3
 ------------|-------|-------|-------
-Presence|No requirements|In-person and unsupervised remote|In-person and supervised remote
-Resolution|No requirements|The minimum attributes necessary to accomplish identity resolution<br><br>KBV may be used for added confidence| Same as IAL2
-Evidence|No identity evidence is collected| - One (1) piece of SUPERIOR or STRONG evidence depending on strength of original proof and validation occurs with issuing source; or<br><br>- Two (2) pieces of STRONG evidence, or <br><br> - One (1) piece of STRONG evidence plus two (2) pieces of FAIR evidence|- Two (2) pieces of SUPERIOR evidence, or<br><br> - One (1) piece of SUPERIOR evidence and one (1) piece of STRONG evidence depending on strength of original proof and validation occurs with issuing source, or<br><br> - Two (2) pieces of STRONG evidence plus one (1) piece of FAIR evidence
-Validation|No validation|Each piece of evidence must be validated with a process that is able to achieve the same strength as the evidence presented|Same as IAL2
+Presence|No requirements|In-person and unsupervised remote.|In-person and supervised remote.
```
Comment (jimfenton) [inline sp800-63a/sec4_ial.md:261]: I tend to put periods in bullet points, tables, etc. only when they're full sentences. Not sure what the standard style is though.

**Review by jimfenton — COMMENTED** [review]
Minor updates.

**Review by jimfenton — CHANGES_REQUESTED** [review]
See above comments (meant to request changes on that review)

File: sp800-63a/sec4_ial.md (hunk 3 above)
Comment (ellenmarie) [reply sp800-63a/sec4_ial.md:261]: I have no preference. Will ask Danna to be sure it's consistent in her WG review.

**Review by jimfenton — APPROVED** [review]

**Pull request merged**